extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -925,6 +925,12 @@ export class AmpLightboxGallery extends AMP.BaseElement {
     let imageAnimation;
 
     const measure = () => {
+      const srcCropEl = closestAncestorElementBySelector(srcImg, 'amp-img');
+      const targetCropEl = closestAncestorElementBySelector(
+        targetImg,
+        'amp-img'
+      );
```

Comment:
(`prettier` is _wild_)

Reply:
Yeah, but maybe we could do better here too. Like does the function name need to be so long such that it wraps even when used with short arguments? If it was just `const targetCropEl = targetImg.closest('amp-img')`, there would be no wrapping.

Perhaps that it forces you to reduce indenting and break things up to avoid the weird behavior might be a good thing?